### ISSUE TITLE
footer links not read as "link" on screen reader

### ISSUE THREAD

**Issue by mark-a-wilson** [bcgov/digital.gov.bc.ca#254]
#### Problem
When using Voice Over on Safari, when I tab through the footer links, the screen reader reads out "Home ...You are currently on a text element." It should say, "Link .... Home".

#### Acceptance Criteria
- [ ] Screen reader reads out footer links as "Link".

**Comment by mark-a-wilson** [comment]
Both our `<nav>` and `<footer>` are marked up in a standard way: The links (`<a>`) are nested within a list item (`<li>`), and all the list items are nested within a list (`<ul>`). 

Though the Voice Over utility for Safari does not announce these links as "Link", the accessibility documentation I have seen seems to say that lists are useful and understandable for assistive technologies. And as I inspect examples of markup for navigation menus and run our page through the [WAVE Web Accessibility Evaluation Tool](https://wave.webaim.org), I cannot see any error in how we are marking up our `<nav>` or `<footer>`.

Going to presume that there's nothing here to fix. Closing this issue.
